src/components/MyAccount/index.tsx

```diff
@@ -95,7 +95,6 @@ const MyAccountContent = ({
   const snackbar = useSnackbar();
   const contentRef = useRef(null);
   const timeZonesEnabled = isEnabled('Timezones');
-  const weightUnitsEnabled = isEnabled('Weight units');
```

Comment:
i can merge main into my branch after you push.. and remove the isEnabled import

Reply:
Ok! I will merge it then